webCommander/centralHub/xTwitter/twitterAPI.py

```diff
@@ -42,12 +47,11 @@ def request_to_x_api(endpoint, headers={}, params={}):
     while response.status_code == TOO_MANY_REQUESTS:
         print('Too many requests, trying in 15 mins...')
         sleep(900)
-        response = request_to_x_api(api_url, headers={}, params=params)
+        response = request_to_x_api(endpoint, headers={}, params=params)
```

Comment:
probably better to do `headers=HEADERS` (caveat of mutable default arguments in python)

<img width="704" alt="image" src="https://github.com/abaddonpuff/commandCenter/assets/387927/c027da68-9eaf-41d5-b6d3-6dd14255660d">